ms2query/ms2library.py

```diff
@@ -0,0 +1,375 @@
+from ms2query.query_from_sqlite_database import get_spectra_from_sqlite, \
+    get_tanimoto_score_for_inchikeys
+from typing import List, Dict, Any
+from matchms.Spectrum import Spectrum
+import pandas as pd
+import numpy as np
+import sqlite3
+import pickle
+from ms2deepscore.models import load_model as load_ms2ds_model
+from ms2deepscore.models import SiameseModel
+from ms2deepscore import MS2DeepScore
+from ms2deepscore.vector_operations import cosine_similarity_matrix
+from gensim.models import Word2Vec
+from tqdm import tqdm
+from spec2vec import SpectrumDocument
+from spec2vec.vector_operations import calc_vector, cosine_similarity_matrix
+from ms2query.ms2query.s2v_functions import \
+    post_process_s2v
+
+# todo Check for _obj.get() for spectrumdocuments and after new release change
+#  to .get()
+
+
+class Ms2Library:
+    def __init__(self,
+                 sqlite_file_location: str,
+                 s2v_model_file_name: str,
+                 ms2ds_model_file_name: str,
+                 pickled_s2v_embeddings_file_name: str,
+                 pickled_ms2ds_embeddings_file_name: str,
```

Comment:
This is starting to become lengths :)
So we might have to do some redesigning here in the near future (but let's make this a different issue).